sign/bls2/bls2.go

```diff
@@ -0,0 +1,150 @@
+// Package bls2 implements the robust BLS scheme that prevents rogue public-key
```

Comment:
I would probably rename this package to something that is more "meaningful" as the 2 in BLS2 doesn't tell me much. Maybe one of `AggregateBLS`, `AggBLS`, `CoBLS`, `MultiSigBLS`, `MultiBLS`, `MBLS` (short form of MultiBLS)?

Reply:
Maybe call it `ASMBLS` for simply `ASM` for Accountable-Subgroup Multi-Signatures as defined in https://eprint.iacr.org/2018/483.pdf ?